codewars_test/test_framework.py

```diff
@@ -102,52 +156,62 @@ def _timed_block_factory(opening_text):
 
     def _timed_block_decorator(s, before=None, after=None):
         display(opening_text, s)
+        is_test_case = opening_text == "IT"
 
         def wrapper(func):
             if callable(before):
                 before()
             time = timer()
+            if is_test_case:
+                func.test_case_func = True
             try:
                 func()
+                if is_test_case:
+                    display("PASSED", "Test Passed")
```

Comment:
This will show extra `<PASSED::>` if any assertion functions were used inside some wrapper.